cyber/time/time.cc

```diff
@@ -54,7 +57,14 @@ Time& Time::operator=(const Time& other) {
   return *this;
 }
 
+void Time::SetNow(const Time& new_now) {
```

Comment:
Better rename this to `SetSimTimeNow` to be more specific.

Reply:
I renamed it to SetSimTime and added comments for it.